<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltroSocios</class>
 <widget class="QWidget" name="FiltroSocios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>541</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#000000</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="filterLineEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #2b2b2b;      
    color: white;                   
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
    selection-background-color: #444444;
    selection-color: #ff4c4c;        
}
</string>
     </property>
     <property name="placeholderText">
      <string>Filtrar por nombre...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">QTableView {
    background-color: #6d6d6d;
    color: white;
    gridline-color: #888888; 
    border: 1px solid #444444;
}

QTableView::item {
    background-color: #7d7d7d;
    border: 1px solid #555555;
    padding: 4px;
}

QTableView {
    background-color: #6d6d6d;
    color: white;
    gridline-color: #888888; /* color del borde entre celdas */
    border: 1px solid #444444;
}

QTableWidget::item {
    background-color: #7d7d7d;
    border: 1px solid #555555;
    padding: 4px;
}

QHeaderView::section {
    background-color: #5d5d5d;
    color: white;
    border: 1px solid #444444;
    padding: 4px;
}

QTableCornerButton::section {
    background-color: #6d6d6d;
    border: 1px solid #555555;
}


QScrollBar:vertical {
    background: #5d5d5d;  /* Fondo liso, sin textura */
    width: 12px;
    margin: 0px;
    border: none;
}

QScrollBar::handle:vertical {
    background: #888888;  /* Parte que se arrastra */
    min-height: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
    background: none;
    border: none;
}

QScrollBar:horizontal {
    background: #5d5d5d;
    height: 12px;
    margin: 0px;
    border: none;
}

QScrollBar::handle:horizontal {
    background: #888888;
    min-width: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    width: 0px;
    background: none;
    border: none;
}

QTableWidget QLineEdit {
    background-color: #5d5d5d;
    color: white;
    border: 1px solid #777777;
    padding: 2px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnModificar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 6px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
       </property>
       <property name="text">
        <string>Modificar Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEliminar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 6px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
       </property>
       <property name="text">
        <string>Eliminar Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 6px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>